src/values/int_value.rs

```diff
@@ -402,6 +405,25 @@ impl IntValue {
         self.int_value.is_const()
     }
 
+    /// Determines whether or not an `IntValue` is an `llvm::ConstantInt`.
+    ///
+    /// ConstantInt only includes values that are known at compile time.
+    ///
+    /// # Example
+    ///
+    /// ```no_run
+    /// use inkwell::context::Context;
+    ///
+    /// let context = Context::create();
+    /// let i64_type = context.i64_type();
+    /// let i64_val = i64_type.const_int(12, false);
+    ///
+    /// assert!(i64_val.is_constant_int());
+    /// ```
+    pub fn is_constant_int(&self) -> bool {
```

Comment:
Could you move this function to be after the other is const fn? Just would like both to be closer together in the docs